Classes/Section Controllers/LoadMore/LoadMoreCell.swift

```diff
@@ -42,14 +49,14 @@ final class LoadMoreCell: UICollectionViewCell {
         set { }
     }
 
-    override var isSelected: Bool {
-        didSet {
-            if isSelected {
-                label.alpha = 0.5
-            } else {
-                label.alpha = 1
-            }
+    
+    func configure(loading: Bool) {
+        label.isHidden = loading
+        if loading {
```

Comment:
you can make this shorter

```swift 
loading ? activity.startAnimating() : activity.stopAnimating()
```

Reply:
I actually just merged this from the IssueNeckLoadMore. Personality I don’t feel ternary vs full if-else makes a difference but I can switch it

Reply:
Functionaly is the same, but is shorter.